### PR TITLE
Navbar tweaks - change navbar item states' font colour & weight

### PR DESCRIPTION
**Pull request by jsms90**
## Top nav tweaks (2, 3 & 4) - [Trello ticket](https://trello.com/c/askfIg4a/448-top-nav-tweaks)

- switch styles for `hover` & `hoverAndActive` states (on both light and dark theme)
- change the styles that we use for "expanded" state of dropdown select button - now based on "hover" rather than "clickTap", so that it looks the same as the anchors next to it
- add bold font weight to active & active+hover

## Review template

- [ ] checked that this PR resolves the spec provided from trello / this description
- [ ] ensured that this PR does not introduce any obvious bugs

### CODE REVIEW

**Comment by antoniocapelo** [comment]
flickr
without having looked at the code, I noticed a flickering of the dropdown items, when hovering with the mouse on the dropdown button (see attached gif)

File: src/components/Header/TopNav/outerItemStates.js [hunk 1]
```diff
@@ -17,12 +17,14 @@ const clickTapLight = css`
 
 const activeLight = css`
   background: ${props => props.theme.colors.white};
-  color: ${props => props.theme.colors.textLight};
+  color: ${props => props.theme.colors.text};
+  font-weight: bold;
```
Comment (antoniocapelo) [inline src/components/Header/TopNav/outerItemStates.js]: I believe this is what's causing the flicker

Comment (antoniocapelo) [reply src/components/Header/TopNav/outerItemStates.js]: or the other *bold* bellow

File: src/components/Header/TopNav/outerItemStates.js [hunk 2]
```diff
@@ -31,8 +33,8 @@ const defaultDark = css`
 `
 
 const hoverDark = css`
-  background: #3a3553;
-  color: ${props => props.theme.colors.white};
+  background: ${props => props.theme.colors.vibrant};
+  color: #007f56;
```
Comment (antoniocapelo) [inline src/components/Header/TopNav/outerItemStates.js]: can't we use a theme var here? :-)

File: src/components/Header/TopNav/outerItemStates.js [hunk 3]
```diff
@@ -17,12 +17,14 @@ const clickTapLight = css`
...
+  color: ${props => props.theme.colors.text};
+  font-weight: bold;
 `
 
 const activeAndHoverLight = css`
-  background: ${props => props.theme.colors.text};
-  color: #a9a9a9;
+  background: ${props => props.theme.colors.greyBG};
+  color: ${props => props.theme.colors.text};
+  font-weight: bold;
```
Comment (antoniocapelo) [inline src/components/Header/TopNav/outerItemStates.js]: IMO, the bold is unnecessary and it really clashes with the rest of the topnav elements :(

File: src/components/Header/TopNav/outerItemStates.js (hunk 2 above)
Comment (jsms90) [reply src/components/Header/TopNav/outerItemStates.js]: Is it worth putting it in the theme, if this is the only place in our codebase where we use this colour?

**Pull request merged**